app/src/main/cpp/Skybox.cpp

```diff
@@ -253,10 +253,19 @@ FileDoesNotExist (const std::string& aName) {
 
 std::string
 Skybox::ValidateCustomSkyboxAndFindFileExtension(const std::string& aBasePath) {
+#if defined(PICOXR) || (defined(OPENXR) && defined(OCULUSVR))
+  auto path = [&](const std::string &name, const std::string &extension) {
+      return aBasePath + "/" + name + "_srgb" + extension;
+  };
+#else
+  auto path = [&](const std::string &name, const std::string &extension) {
+      return aBasePath + "/" + name + extension;
+  };
```

Comment:
maybe we could have a single path method and just ifdef the code that selects the format